package/contents/config/main.xml

```diff
@@ -71,4 +75,4 @@
             <label>unhighlighted color</label>
         </entry>
     </group>
-</kcfg>
```

Comment:
is this intentional?

Reply:
Removing this empty line is done by my editor and it does not result any noticeable issue, so I leave it here instead.